DEFI/ERC20/certora/specs/ERC20Full.spec

```diff
@@ -0,0 +1,785 @@
+
+/***
+This example is a full spec for erc20.
+To run this use Certora cli with the conf file runERC20Full.conf
+Example of a run: https://vaas-stg.certora.com/output/1512/874bac3d0eec43c6824ff9910f884e98?anonymousKey=b36ddee07d44f489a78330a5f994420b693c9a57
+Mutation test for this spec: https://mutation-testing-beta.certora.com?id=c397b634-0f66-4e71-8946-903a378085cd&anonymousKey=ee7c435e-c696-4e8a-8836-988aa395311d
+See https://docs.certora.com for a complete guide.
+***/
```

Comment:
nice, can you also have the above comment in a readme.md file. 
rerun on cloud - staging links are not supported on version changes so it will break